event.go

```diff
@@ -262,6 +262,11 @@ func (eb *EventBuilder) Build(
 // It also checks the content hashes to ensure the event has not been tampered with.
 // This should be used when receiving new events from remote servers.
 func NewEventFromUntrustedJSON(eventJSON []byte, roomVersion RoomVersion) (result Event, err error) {
+	if r := gjson.GetBytes(eventJSON, "_*"); r.Exists() {
+		err = UnexpectedHeaderedEvent{}
```

Comment:
Can we give context here (mention `NewEventFromUntrustedJSON`?)

Reply:
You mean in the actual error text?